spec/controllers/master/profiles_controller_spec.rb

```diff
@@ -66,7 +66,7 @@
     it 'prevents removing last account' do
       user = FactoryGirl.create(:master_with_vk_account)
       delete :remove_account, {format: :json, id: user.accounts.first.id}
-      expect(JSON.parse(response.body)['error']).to eq 'you cannot remove last account'
+      expect(JSON.parse(response.body)['error']).to eq 'Вы неможете удалить единственный аккаунт'
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.